src/pipeline/src/etl/error.rs

```diff
@@ -363,6 +363,12 @@ pub enum Error {
         #[snafu(implicit)]
         location: Location,
     },
+    #[snafu(display("Wrong digest pattern: {pattern}"))]
+    PresetPatternInvalid {
```

Comment:
Not only preset patterns use this error, regex patterns also use.
```suggestion
    DigestPatternInvalid {
```